azure-iptables-monitor/iptables_monitor.go

```diff
@@ -291,6 +338,28 @@ func main() {
 			}
 		}
 
+		// if disabled the number of blocks never increases from zero
+		currentBlocks := uint64(0)
+		if *checkMap {
+			// read bpf map to check for number of blocked iptables rules
+			currentBlocks, err = getBPFMapValue()
+			if err != nil {
+				klog.Errorf("failed to get bpf map value: %v", err)
+			}
+			klog.V(2).Infof("IPTables rules blocks: Previous: %d Current: %d", previousBlocks, currentBlocks)
+		}
+		// if number of blocked rules increased since last time
+		blockedRulesIncreased := currentBlocks > previousBlocks
+		if *sendEvents && blockedRulesIncreased {
+			msg := fmt.Sprintf("Blocked iptables rules increased from %d to %d since last check. "+
+				"IPTables rules blocked because EBPF Host Routing is enabled: aka.ms/acnsperformance", previousBlocks, currentBlocks)
```

Comment:
Nit: Let's use "Iptables". in other places, we're saying "iptables".

Reply:
changed to "iptables" since we previously referenced it in lowercase